scheduler/clients/javascript/lib/userClient.ts

```diff
@@ -27,14 +28,34 @@ type GetUserFeebusyReq = {
   calendarIds?: string[]
 }
 
+/**
+ * Request to get multiple users' freebusy status
+ */
+type GetMultipleUsersFeebusyReq = {
+  /**
+   * List of user ids to check for freebusy
+   */
+  userIds: UUID[]
+  /**
+   * Start time of the period to check for freebusy
+   * @format Date in UTC
```

Comment:
I assume you've thought of this so maybe just catch me up, but:

my expectation as a user of the client would be that I'd be able to query with timezone information, this seems like the main place someone would make a mistake there.

Is there a reason why this expects UTC?

Reply:
I have started looking into a bit more in https://github.com/meetsmore/nittei/pull/28, following the article that Derek shared, but it's a bit more complex than expected (I need to invest more time).

Ideally, I would like (at least) the JS SDK, the Rust SDK and the API to accept any timezone.
The API could still only handle only UTC datetimes internally. In the PR above, I've tried to have timezones everywhere (so, never convert to UTC, but it became quite complex so I decided to focus on the other requirements as this one is not blocking per se